meson.build

```diff
@@ -0,0 +1,78 @@
+project('dstep', 'd', version: '1.0.1')
+
+if get_option('buildtype') == 'release'
+    add_project_arguments('-O3', language: 'd')
```

Comment:
This is wrong. A project should never make this kind of decisions if not required.

Reply:
It has been removed.